lms/djangoapps/certificates/api.py

```diff
@@ -623,6 +631,52 @@ def get_allowlist_entry(user, course_key):
     return allowlist_entry
 
 
+def is_on_allowlist(user, course_key):
+    """
+    Determines if a learner has an active allowlist entry for a given course-run.
+    """
+    log.info(f"Checking if student {user.id} is on the allowlist in course {course_key}")
```

Comment:
Small question, are we potentially over logging in some of the new methods? This is not a blocker.

Reply:
I didn't think so. I thought it would all be useful information to have if we ever had to troubleshoot issues with the allowlist since we are touching lots of old code. If you feel strongly about this I can remove some of it. I didn't think the allowlist feature was used enough that it would be spamming/filling logs with useless info.

Let me know if you have any other concerns I may not be thinking of!